src/util.h

```diff
@@ -32,12 +32,17 @@
 
 #include <stdint.h>
 #include "sds.h"
+#include <fcntl.h>
 
 /* The maximum number of characters needed to represent a long double
  * as a string (long double has a huge range).
  * This should be the size of the buffer given to ld2string */
 #define MAX_LONG_DOUBLE_CHARS 5*1024
 
+#define REDIS_NONBLOCK_PIPE O_NONBLOCK
+#define REDIS_CLOEXEC_PIPE O_CLOEXEC
+#define REDIS_DEFAULT_PIPE O_CLOEXEC|O_NONBLOCK
```

Comment:
Better be explicit and write O_CLOEXEC|O_NONBLOCK everywhere. It makes the code more clear IMO.

Reply:
Done.